src/partitioned_data/partitionedNLPModels/partitionedNLPModel.jl

```diff
@@ -76,4 +76,70 @@ function NLPModels.grad!(
   return g
 end
 
+"""
+    B = hess_approx(nlp::PartiallySeparableNLPModel)
+
+Return the Hessian approximation of `nlp`.
```

Comment:
which approximation?

Reply:
When we release the v0.1.0, there was no `hess` method.
`hess_approx` returns the current state of the partitioned matrix.
It is not a new function, but I moved it alongside NLPModel methods.

Reply:
But I don't understand what approximation we are talking about. A PQN approximation?

Reply:
Yes.
It returns the partitioned matrix which is updated by partitioned quasi-Newton methods.
We will delete it when we'll move out the PQN stuffs.